azurerm/internal/services/network/tests/resource_arm_network_interface_test.go

```diff
@@ -444,6 +469,66 @@ resource "azurerm_network_interface" "test" {
 `, template, data.RandomInteger)
 }
 
+func testAccAzureRMNetworkInterface_withMultipleParameters(data acceptance.TestData) string {
+	template := testAccAzureRMNetworkInterface_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_network_interface" "test" {
+  name                    = "acctestni-%d"
+  location                = azurerm_resource_group.test.location
+  resource_group_name     = azurerm_resource_group.test.name
+  enable_ip_forwarding    = true
+  internal_dns_name_label = "acctestni-%s"
+
+  dns_servers = [
+    "10.0.0.5",
+    "10.0.0.6"
+  ]
+
+  ip_configuration {
+    name                          = "primary"
+    subnet_id                     = azurerm_subnet.test.id
+    private_ip_address_allocation = "Dynamic"
+  }
+
+  tags = {
+    env = "Test"
+  }
+}
+`, template, data.RandomInteger, data.RandomString)
+}
+
+func testAccAzureRMNetworkInterface_updateMultipleParameters(data acceptance.TestData) string {
+	template := testAccAzureRMNetworkInterface_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_network_interface" "test" {
+  name                    = "acctestni-%d"
+  location                = azurerm_resource_group.test.location
+  resource_group_name     = azurerm_resource_group.test.name
+  enable_ip_forwarding    = true
+  internal_dns_name_label = "acctestni-%s"
+
+  dns_servers = [
+    "10.0.0.5",
+    "10.0.0.6"
+  ]
```

Comment:
The linked issue refers to changing DNS servers, but these stay the same through the update test, should we be testing/validating this in the update?

Reply:
updated